data-reporting-service/observation-service/src/main/java/gov/cdc/etldatapipeline/observation/controller/ObservationController.java

```diff
@@ -9,7 +9,7 @@
 @RestController
 public class ObservationController {
     private final KafkaProducerService producerService;
-    private final String topicName = "cdc.nbs_odse.dbo.Observation";
+    private final String topicName = "nbs_Observation";
```

Comment:
This can be configured in the application.yaml

Reply:
I'll move it